egs2/TEMPLATE/mt1/conf/fbank.conf

```diff
@@ -0,0 +1,2 @@
+--sample-frequency=16000 
```

Comment:
Is this file necessary?

Reply:
Thanks I removed it and now just rechecking the entire pipeline!